django/translation.md

```diff
@@ -0,0 +1,129 @@
+# Translating Django apps
+
+This document describes our best practices for translating Django apps.
+
+The Django documentation for translation (or, in technical terms,
+[internationalization and localization](https://docs.djangoproject.com/en/3.0/topics/i18n/))
+is quite extensive and we encourage you to read it if you plan to translate a
+Django app. This document will not explain in detail how translation works in
+Django, so think of it as a supplement to the official docs.
+
+## Contents
+
+- [Internationalization](#internationalization)
+    - [Configure Django settings for translations](#configure-django-settings-for-translations)
+    - [Tag strings for translation](#tag-strings-for-translation)
+- [Localization](#localization)
+    - [Install Rosetta](#install-rosetta)
+    - [Make message files](#make-message-files)
+    - [Keep message files under version control](#keep-message-files-under-version-control)
+    - [Compile messages on build and deploy](#compile-messages-on-build-and-deploy)
+    - [Automatically reload catalog files during translation](#automatically-reload-catalog-files-during-translation)
+- [Exmaples](#examples)
+
+## Internationalization
+
+Internationalization refers to the process of setting up an application for
+translation into multiple langauges. Internationalization is typically performed
+by the developers of an app as a separate step from the translation itself.
+
+### Configure Django settings for translations
+
+Configure the following settings:
```

Comment:
Can you change these later?

Reply:
Yes! I just pushed https://github.com/datamade/how-to/pull/106/commits/6b7e04b0e54faf77694fb86861287b540002bb8d to clarify that these settings can change. Does that answer your question or do you think it needs more detail?

Reply:
Nope, that's perfectly fine. Thanks, @jeancochrane!